README.md

```diff
@@ -1,7 +1,28 @@
-# notation-core-go
+# Notation-core-go library
 
 [![Build Status](https://github.com/notaryproject/notation-core-go/actions/workflows/build.yml/badge.svg?event=push&branch=main)](https://github.com/notaryproject/notation-core-go/actions/workflows/build.yml?query=workflow%3Abuild+event%3Apush+branch%3Amain)
 [![codecov](https://codecov.io/gh/notaryproject/notation-core-go/branch/main/graph/badge.svg)](https://codecov.io/gh/notaryproject/notation-core-go)
 [![Go Reference](https://pkg.go.dev/badge/github.com/notaryproject/notation-core-go.svg)](https://pkg.go.dev/github.com/notaryproject/notation-core-go@main)
 
-Contains support for Notary Project signature envelope, and format specific implementation
+Notation-core-go library provides specific implementation for Notary Project signature envelope, signature format, and OCSP revocation check.
+
+Notation-core-go library reaches stable in July 2023 and is in active development status.
+
+> **Note** The Notation documentation is available [here](https://notaryproject.dev/docs/). You can also find the Notary Project [README](https://github.com/notaryproject/.github/blob/main/README.md) to learn about the overall Notary Project.
+
+## Table of Contents
+- [Core Documents](#core-documents)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+ 
+## Core Documents
+
+All library documentation is available at [Go Reference](https://pkg.go.dev/github.com/notaryproject/notation-core-go).
+
+## Code of Conduct
```

Comment:
Suggest referencing to `https://github.com/notaryproject/.github/blob/main/CODE_OF_CONDUCT.md`, which was updated months ago per TOC review.